hastexo/tasks.py

```diff
@@ -99,7 +107,8 @@ def launch_stack(self, heat, stack_name, stack_template):
             status = stack.stack_status
             retry += 1
             if retry >= self.retries:
-                status = 'CREATE_FAILED'
+                status_prefix = status[:-12]
```

Comment:
-12 is to remove the suffix from *_IN_PROGRESS, the only possible statuses here.  But ok, a regex sub will do just as well.